source/_patterns/utility-classes/_alignment.scss

```diff
@@ -24,6 +24,11 @@
   margin-right: auto;
 }
 
+// Breaks something outside its parent l-constrain or other centered container.
+.u-align-full {
+  @include layout-full-bleed();
+}
```

Comment:
I don’t this utility class has anything to do with alignment. I’d prefer if we kept the old `u-full-width` class.

Reply:
I don't necessarily disagree. But since WordPress has an internal class called alignfull (to go along with alignleft, alignright, etc), that sort of pushed me to think "Well, it kind of IS about alignment (to the screen anyway) and we already have an alignment file and this would make it easier to be consistent with WordPress' categorization and class names by consolidating." 

I'm a bit torn. 

@timnolte @kmonahan @tomealter or others -- any strong feelings one way or the other? If we kept it as full-width, on the WP side I'd probably apply the mixin to both classes (full-width and alignfull).

Reply:
I don't know about _strong_ feelings, but I lean toward the second option: keep the utility class name `u-full-width` and also apply the mixin to the WP-specific class in gesso-wp.

Reply:
Well so I guess my hangup is that WordPress also has `alignwide` which should not be full width. It should be wider than the main content area, but not as wide as the full alignment. And I would argue that this is alignment related since even in a desktop text/document editor such as Word making text full width(going from edge-to-edge between the page margins) is an alignment formatting.

IMO I want the theme to be following what is compatible and a standard for the CMS. When we go against the CMS then we can't use out-of-the-box features and it's a maintainability pain. And I don't want to reinvent the wheel because something non-WordPress is dictating how we use the CMS.

Reply:
If we still account for what WordPress needs then I'm fine with the mixin naming so long as we get the right support out-of-the-box from Gesso/PL.

Reply:
Ok, I've reverted this to the `u-full-width` class.  I'll create a similar PR to the WordPress side of things that will be the same, except I will also apply the same mixin to the `alignfull` class.

Reply:
@dcmouyard @kmonahan I think this is ready to merge in, if you're good with it.  I want to get your thoughts on the addition I made to the `l-site-container` class though.  I realized during testing that the full bleed class creates a horizontal scroll bar (at least in some browsers and OSs).  The article Tim listed above suggests adding `width: 100%; overflow-x: hidden;` to a site wrapper, as did most of the pages I found talking about this full width approach.

Reply:
There's another approach we used on EPA, where you set the value of a CSS variable with JS to account for the width of the vertical scrollbar (which is what typically causes the overflow with 100vw). Are we trying to keep Gesso IE11 compatible?

Reply:
I have been using the `l-site-container` component to place max-widths on my last couple of sites for the whole page (including full-bleed components). This is to avoid reeeally wide full-bleed components on very wide monitors. I spun up a quick example in codepen and placing the `overflow-x: hidden;` appears to work like a charm 👍 
https://codepen.io/tomealter/pen/yLXyBoW?editors=1100

Reply:
Thanks, @tomealter!

@kmonahan I'd rather not invite JS to the party unless CSS can't handle hosting the festivities alone.  I've used the overflow trick before and rarely run into issues.  Biggest issue might be really wide tables that would normally overflow the site and force a scrollbar.  But now that I think about it, we should probably have a responsive table fix in place out-of-the-box.  I'll create a separate PR for that.